### PR TITLE
rkrlv2: beta_2 -> beta_3

### PR DESCRIPTION
**Pull request by siraben**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 108043` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>rkrlv2</li>
  </ul>
</details>

**Review by prusnak — CHANGES_REQUESTED** [review]
Seems this is broken on aarch64 - https://github.com/NixOS/nixpkgs/pull/108043/checks?check_run_id=1630340467

Can you please add the following to `meta`?

```
    broken = stdenv.isAarch64; # g++: error: unrecognized command line option '-mfpmath=sse'
```

File: pkgs/applications/audio/rkrlv2/default.nix [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+  buildInputs = [ libXft libXpm lv2 fftw libjack2 libsamplerate libsndfile ];
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "Rakarrak effects ported to LV2";
     homepage = "https://github.com/ssj71/rkrlv2";
-    license = stdenv.lib.licenses.gpl3;
-    maintainers = [ stdenv.lib.maintainers.joelmo ];
-    platforms = stdenv.lib.platforms.linux;
+    license = licenses.gpl2;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/rkrlv2/default.nix:21]: Only or Plus?

**Comment by siraben** [comment]
> Can you please add the following to `meta`?

Done

**Review by prusnak — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108043` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>rkrlv2</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

rkrlv2.log:
<details>
`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/applications/audio/rkrlv2/default.nix:21:5:

```
   |
21 |     license = licenses.gpl2Only;
   |     ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md
</details>

**Comment by siraben** [comment]
>  `gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

resolved.

**Comment by SuperSandro2000** [comment]
> > `gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.
> 
> resolved.

That is an error with nixpkgs-hammer and is hopefully filtered out in the future.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 108043` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>rkrlv2</li>
  </ul>
</details>

**Pull request merged**